source/includes/steps-fle-convert-to-a-remote-master-key-kmip.yaml

```diff
@@ -0,0 +1,126 @@
+title: Configure Your KMIP KMS
+ref: configure-kmip-kms
+content: |
+
+  To connect a MongoDB driver client to your KMIP KMS, you must configure your KMS
+  such that it has the following behaviors:
+
+  - Accepts your client's TLS certificate
+  - Contains a symmetric master key
```

Comment:
A master key is not required. If a `keyId` is not included in the `masterKey` document, the driver will create a new "master key".

The "master key" is a 96 byte SecretData Managed Object. Those are terms specific to the [KMIP specification](http://docs.oasis-open.org/kmip/spec/v1.4/os/kmip-spec-v1.4-os.html#_Toc490660773). 

I suggest:
- Remove this bullet
- Or say that a master key may be preconfigured. If so, it must be a 96 byte SecretData Manage Object.

Reply:
I have removed the bullet and added a note to the "keyId" field option describing the following behavior:

> If you do not specify the ``keyId`` field in the ``masterKey`` document
   you send to your KMIP KMS, the driver creates a new
   96 Byte Secret Data managed object in your KMS to act as your
   master key.

Reply:
Nice, that description SGTM.